docgen/src/guide/Multi_index.md

```diff
@@ -6,22 +6,19 @@ category: guide
 navWeight: 50
 ---
 
-You can use multiple [`<InstantSearch>`](guide/<InstantSearch>.html) instances for cases like:
+Whenever you want to:
 
-* displaying hits from different indices
-* sharing a single SearchBox
-* any use case involving synchronizing widgets between different [`<InstantSearch>`](guide/<InstantSearch>.html) instances
+* display hits from different indices
+* share a single SearchBox
+* synchronize widgets between different indices
 
-Two props on the [InstantSearch component](widgets/InstantSearch.html) can be used to inject searchState or be notified of searchState changes:
+You can use multiple [`<MultiIndexContext>`](widgets/<MultiIndexContext>.html) instances.
```

Comment:
I proposed IndexContext in another comment, we could even go for Index indeed, very short and still valid.